crates/ty_wasm/src/lib.rs

```diff
@@ -421,7 +421,7 @@ impl Workspace {
             .into_iter()
             .map(|completion| Completion {
                 kind: completion.kind(&self.db).map(CompletionKind::from),
-                name: completion.name.into(),
+                name: completion.inner.name.into(),
```

Comment:
It would be nice to also hook up documentation in the playground. You have to add a new `documentation` field to `Completion`, then propagate the field here

https://github.com/astral-sh/ruff/blob/8c0743df97dc7afa1deab7506b322551f2e392bb/playground/ty/src/Editor/Editor.tsx#L321

Reply:
I think done? Never tested the playground.